src/utils/CalendarUtils/CalendarUtils.js

```diff
@@ -51,3 +51,20 @@ export const elementExists = selector => {
 
   return javaScriptUtils.isDefined(element);
 };
+
+export const filterContributionDays = (dailyData, from) => {
```

Comment:
`from` doesn't really sound like "it's the start of the timeframe", does it? Proposals welcome.

Reply:
`startDate`?

Reply:
👍

Reply:
fc82179